TimoCloud-Universal/src/main/java/cloud/timo/TimoCloud/common/sockets/MessageHandler.java

```diff
@@ -6,14 +6,24 @@
 
 public abstract class MessageHandler {
     private final MessageType messageType;
+    private final boolean handShake;
 
     public MessageHandler(MessageType messageType) {
+        this(messageType, false);
+    }
+
+    public MessageHandler(MessageType messageType, boolean handShake) {
         this.messageType = messageType;
+        this.handShake = handShake;
     }
 
     public abstract void execute(Message message, Channel channel);
 
     public MessageType getMessageType() {
         return messageType;
     }
+
+    public boolean isHandShake() {
```

Comment:
Rename to `needsHandshake()` oder `usesHandshake()` or `requiresHandshake()`

Add auch simple JavaDoc explaining what this property means,.